news/4749.hooks.rst

```diff
@@ -0,0 +1 @@
+Fix shapely hook on Windows for non-conda shapely installations.
```

Comment:
As this shall close #2834, please *copy* this file to 2834.… to make the release process create the appropriate reference.

Reply:
Done